File: Data Structures & Algorithms/sort-array-by-parity/submission-0.py
class Solution:
    def sortArrayByParity(self, nums: List[int]) -> List[int]:
        i, j = 0, len(nums)-1
        while i < j:
            if nums[i] % 2 == 0:
                i += 1
            else:
                nums[j], nums[i] = nums[i], nums[j]
                j -=1
        return nums

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set numSet(nums.begin(),nums.end());
        int longest = 0;

        for (int x: numSet) {
           if (numSet.find(x-1) == numSet.end()) {
            int curr = 1;
            while (numSet.find(x+curr) != numSet.end()) curr++;
            longest = max(longest, curr);
           } 
        }
        return longest;

    }
};
